### DISCUSSION TITLE
Selecting text in code diff highlights leaves the first 1 or 2 characters behind, without selecting them

### DISCUSSION THREAD

**Discussion by rogerluan** [community/community#57708] [Repositories]
### Select Topic Area

Bug

### Body

In Safari:

<img width="520" alt="image" src="https://github.com/community/community/assets/8419048/19a71b02-9dfb-4846-b09f-ef780dab3430">

## Steps to reproduce

1. Open a PR, and go to the diff section where you can see the code side by side
2. Select an line using your mouse. Remember: you intend to select the entire line — all of its characters!
3. Observe that the first 1 or 2 characters are left behind as soon as you're done highlighting what was meant to be the entire line.

## Demo

![CleanShot 2023-06-10 at 21 28 16](https://github.com/community/community/assets/8419048/f6ee81e8-b47a-4e03-bd07-c6110cb77081)

The same happens if you select from left to right as well.

**Comment by Desareollo** [comment]
La selección es así por qué si lo seleccionará todo se descuadra  ya que es algo que no tienen seguido o finalizó sin terminal.

**Reply by rogerluan** [reply]
I don't understand your explanation to justify this behavior 😬 this used to work before, so it's a regression, the way I see it. It's been like this for like 1-2 weeks tops (from the moment I noticed, of course)
